src/client/components/Header.tsx

```diff
@@ -39,10 +61,43 @@ export const Header: React.FunctionComponent = (props) => {
             ? null :
               isLoggedIn
                 ? [
-                    <IconButton color="inherit" key={0}><SettingsIcon color="inherit"/></IconButton>,
+                    <IconButton color="inherit" key={0} onClick={() => setOpen(true)}><SettingsIcon color="inherit" /></IconButton>,
                     <Fragment key={1}>{data.me.name}</Fragment>,
-                    <Button onClick={logOutUser} key={2}>Log Out</Button>
+                    <Button onClick={logOutUser} key={2}>Log Out</Button>,
+                    <Modal open={open} style={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      justifyContent: 'center',
+                    }}>
```

Comment:
This is a component in an array, so you should add a key to this, which you can have just be 3. This is important for how react works, you can see the error message here: 
![image](https://user-images.githubusercontent.com/52944725/99485786-c5dda500-2928-11eb-809d-aeaf0745e54d.png)

when you see that error message, you need to add Unique keys to all of the objects in the array.